integration_tests/conftest.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+import pytest
+from requests import HTTPError
+
+import datareservoirio as drio
+
+
+@pytest.fixture(autouse=True, scope="session")
+def set_environment_qa():
+    """Set environment to 'QA'"""
+    env = drio.environments.Environment()
+    env.set_qa()
+
+
+@pytest.fixture(scope="session")
```

Comment:
The test fails at setup, says client_id and client_secret are "None"

Reply:
You have to set the environment variables before running the test